src/NuGet.Core/NuGet.Versioning/NuGetVersion.cs

```diff
@@ -184,5 +184,13 @@ public int Revision
         {
             get { return _version.Revision; }
         }
+
+        /// <summary>
+        /// Returns true if version is a SemVer 2.0.0 version
+        /// </summary>
+        public bool IsSemVer2
+        {
+            get { return this.ReleaseLabels.Count() > 1 || this.HasMetadata; }
```

Comment:
Yes, that is the way it works.  A second ReleaseLabel means it's using version 2.0.0.